src/library/simplified/extension/currency-code.js

```diff
@@ -90,7 +90,8 @@ else if(utag.isLXGT_CO() && b.entity.checkout.cartTotal.currency){
     b['currencyCode'] = b.entity.checkout.cartTotal.currency;
 }
 
-//All Slim confirmation pages
-if(b.pageInfo.pageName && b.pageInfo.pageName.indexOf("Checkout.Confirmation.Slim") > -1 && b.entity.checkout.currency){
+//All confirmation pages in cluding Slim confirmation pages.
+if(b.pageInfo.pageName && b.pageInfo.pageName.indexOf("Checkout.Confirmation") > -1 && b.entity.checkout.currency){
```

Comment:
why all confirmation pages? I see many specific logic for different LOB: isLXCO, isCarCO, isLXGT_CO

Reply:
Not all cko pages have b.entity.checkout.currency, but most of the new uCKO pages have b.entity.checkout.currency.

Reply:
Could we use ‘else if’ at least? Because looks like this if will always get evaluated